### PR TITLE
💄✏️ Clm 499 - Settings - Company Info

### PR DESCRIPTION
**Pull request by fraciah**
# Description

Fix channel's platform typo and logo appearance.

# Screenshots 
b4
after

Capture
Capture2

### CODE REVIEW

File: libs/private/features/convs-mgr/settings/organisation-details/src/lib/components/company-data/company-data.component.html [hunk 1]
```diff
@@ -120,7 +120,7 @@ <h3 class="channel-head">My Channels</h3>
         
         <ng-container matColumnDef="type">
           <th mat-header-cell *matHeaderCellDef> {{"COMPANY-INFORMATION.CHANNEL-MODAL.PLATFORM" | transloco}}</th>
-          <td mat-cell *matCellDef="let element"> {{element.type}} </td>
+          <td mat-cell *matCellDef="let element"> {{transformChannelPlatform(element.type)}} </td>
```
Comment (jrosseel) [inline libs/private/features/convs-mgr/settings/organisation-details/src/lib/components/company-data/company-data.component.html:123]: Please use a local pipe

Comment (jrosseel) [reply libs/private/features/convs-mgr/settings/organisation-details/src/lib/components/company-data/company-data.component.html:123]: https://v17.angular.io/guide/pipes-overview

Comment (jrosseel) [reply libs/private/features/convs-mgr/settings/organisation-details/src/lib/components/company-data/company-data.component.html:123]: Local means declared in the local module/package

**Review by jrosseel — CHANGES_REQUESTED** [review]

File: libs/private/features/convs-mgr/settings/organisation-details/src/lib/pipes/platform.pipe.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { Pipe, PipeTransform } from '@angular/core';
```
Comment (ReaganCn) [inline libs/private/features/convs-mgr/settings/organisation-details/src/lib/pipes/platform.pipe.ts:1]: Check my comments in CLM - 490

**Review by ReaganCn — CHANGES_REQUESTED** [review]

**Pull request merged**